camayoc/tests/qcs/cli/test_scans.py

```diff
@@ -0,0 +1,332 @@
+    qpc_scan_start = pexpect.spawn(command)
+    qpc_scan_start.logfile = io.BytesIO()
+    assert qpc_scan_start.expect(pexpect.EOF) == 0
+    qpc_scan_start.close()
+    output = qpc_scan_start.logfile.getvalue().decode('utf-8')
+    assert qpc_scan_start.exitstatus == exitstatus, output
+    return output
+
+
+def scan_show(options=None, exitstatus=0):
+    command = 'qpc scan show'
+    for key, value in options.items():
+        if value is None:
+            command += ' --{}'.format(key)
+        else:
+            command += ' --{}={}'.format(key, value)
+    qpc_scan_show = pexpect.spawn(command)
+    qpc_scan_show.logfile = io.BytesIO()
+    assert qpc_scan_show.expect(pexpect.EOF) == 0
+    qpc_scan_show.close()
+    output = qpc_scan_show.logfile.getvalue()
+    assert qpc_scan_show.exitstatus == exitstatus, output.decode('utf-8')
+    return json.loads(output)
+
+
+def scan_pause(options=None, exitstatus=0):
+    command = 'qpc scan pause'
+    for key, value in options.items():
+        if value is None:
+            command += ' --{}'.format(key)
+        else:
+            command += ' --{}={}'.format(key, value)
+    qpc_scan_pause = pexpect.spawn(command)
+    qpc_scan_pause.logfile = io.BytesIO()
+    assert qpc_scan_pause.expect(pexpect.EOF) == 0
+    qpc_scan_pause.close()
+    output = qpc_scan_pause.logfile.getvalue().decode('utf-8')
+    assert qpc_scan_pause.exitstatus == exitstatus, output
+    return output
+
+
+def scan_cancel(options=None, exitstatus=0):
+    command = 'qpc scan cancel'
+    for key, value in options.items():
+        if value is None:
+            command += ' --{}'.format(key)
+        else:
+            command += ' --{}={}'.format(key, value)
+    qpc_scan_cancel = pexpect.spawn(command)
+    qpc_scan_cancel.logfile = io.BytesIO()
+    assert qpc_scan_cancel.expect(pexpect.EOF) == 0
+    qpc_scan_cancel.close()
+    output = qpc_scan_cancel.logfile.getvalue().decode('utf-8')
+    assert qpc_scan_cancel.exitstatus == exitstatus, output
+    return output
+
+
+def scan_restart(options=None, exitstatus=0):
+    command = 'qpc scan restart'
+    for key, value in options.items():
+        if value is None:
+            command += ' --{}'.format(key)
+        else:
+            command += ' --{}={}'.format(key, value)
+    qpc_scan_restart = pexpect.spawn(command)
+    qpc_scan_restart.logfile = io.BytesIO()
+    assert qpc_scan_restart.expect(pexpect.EOF) == 0
+    qpc_scan_restart.close()
+    output = qpc_scan_restart.logfile.getvalue().decode('utf-8')
+    assert qpc_scan_restart.exitstatus == exitstatus, output
+    return output
+
+
+def wait_for_scan(scan_id, status='completed', timeout=900):
+    """Wait for a scan to reach some ``status`` up to ``timeout`` seconds.
+
+    :param scan_id: Scan ID to wait for.
+    :param status: Scan status which will wait for. Default is completed.
+    :param timeout: wait up to this amount of seconds. Default is 900.
```

Comment:
might want to be specific about time units, I think it is seconds.

Reply:
It says `wait up to this amount of seconds`, I am wondering if you are mentioning the unit for the default value later?

Reply:
My bad. Not my expected word order but I think it is fine.

Reply:
What would be a better word order?

Reply:
I think I was just reading too fast, because the fact that it starts with `:param timeout:` means we are defining that argument.